src/construction/casting.rs

```diff
@@ -15,9 +18,7 @@ impl ApInt {
     ///
     /// - If the `target_width` is greater than the current width.
     pub fn truncate<W>(&mut self, target_width: W) -> Result<()>
-    where
-        W: Into<BitWidth>,
-    {
+    where W: Into<BitWidth> {
```

Comment:
This `where` clause formatting is inferior to the previous one when it comes to easily edit a where clause which happens frequently during prototyping of new code. I am really a fan of the old formatting because of this reason.

Reply:
ok